webui/src/components/Service.jsx

```diff
@@ -273,6 +277,9 @@ class Service extends React.Component {
                             <button type="submit" className="btn btn-default" onClick={this.handleModalOpen.bind(this)}>
                                 <span className="glyphicon glyphicon-edit" aria-hidden="true"></span> Edit Metadata
                             </button>
+                            <button type="submit" className="btn btn-default" onClick={this.handleRemoveService.bind(this)}>
```

Comment:
The button type here and above should not be 'submit', but 'button', as there is no form involved.

Reply:
right